docs/destinations/forwarding-events/event-forwarding-monitoring-and-troubleshooting/index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "Monitoring and troubleshooting forwarders"
+description: "Monitor event forwarder performance, debug failures, and understand retry logic with cloud metrics, failed event logs, and BDP Console statistics."
+sidebar_position: 15
+---
+
+This page is an overview of how to monitor event forwarder performance and diagnose delivery issues. Snowplow provides both summary metrics and detailed failed event logs, to help you understand failure patterns and troubleshoot specific problems.
+
+## Failure types and retry logic
+
+Forwarders uses the same retry logic and failure handling as the underlying [Snowbridge failure model](/docs/api-reference/snowbridge/concepts/failure-model/index.md). Snowplow handles event forwarding failures differently depending on the type:
```

Comment:
Said on call but commenting for reference: I would remove the dependency on api docs and just describe the failure behaviours as below, which are already quite good!

Won't die on this hill but think it's the wiser move